README.md

```diff
@@ -66,6 +67,48 @@ module.exports = {
 };
 ```
 
+#### Frequently-Disabled Rules
+
+Some rules that are useful for catching mistakes in regular source code are counterproductive to clarity and brevity in code snippets and examples.
+For example `no-undef` might flag uses of variables that code examples assume are already in scope, and declaring or importing them in every code block would distract from the point of each example.
```

Comment:
```suggestion
For example, in snippets, `no-undef` might flag variables that are declared outside of the displayed code, because declaration is non-essential to the example being showcased.
```